bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -247,6 +256,19 @@ protected void initialize(AbstractConfiguration<?> conf,
             acls);
     }
 
+    private synchronized void handleState(Watcher.Event.KeeperState zkClientState) {
+        switch (zkClientState) {
+            case Expired:
+            case Disconnected:
+                this.metadataServiceAvailable = false;
+                break;
+
```

Comment:
nit: remove line break

Reply:
@congbobo184 looks like line break is not removed?

Reply:
@congbobo184 line 256 blank line?

Reply:
sorry, I thought it was to delete break